uktrade/data-hub-frontend
# Move <FieldCheckboxes> to Data Hub FE and create filter version.

## Description of change
As part of the work around moving Data Components to Data Hub FE this one ports the `<FieldCheckboxes>` and creates a reduced version of it so we can use it as a filter in the collection lists page.

Note: you will see a small change to the input field, this is because I made some changes around how the <FieldWrapper> effects child elements such as input.

## Test instructions

1. Run storybook and you should see the checkboxes available in the navigation.
2. Check that the reduced versions styles are inline with what we currently on Data Hub.

## Screenshots
### Before
![Screenshot 2020-07-31 at 08 54 45](https://user-images.githubusercontent.com/10154302/89013483-ddf2cf80-d30b-11ea-9790-664c3897da56.png)
![Screenshot 2020-07-31 at 08 54 52](https://user-images.githubusercontent.com/10154302/89013495-e21eed00-d30b-11ea-8fb5-fab711e7294e.png)
![Screenshot 2020-07-31 at 08 55 00](https://user-images.githubusercontent.com/10154302/89013499-e64b0a80-d30b-11ea-9819-33519c84521f.png)


## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
